src/test/rgw/rgw_multi/tests.py

```diff
@@ -92,15 +92,21 @@ def datalog_status(zone):
 def datalog_autotrim(zone):
     zone.cluster.admin(['datalog', 'autotrim'])
 
-def bilog_list(zone, bucket, args = None):
+def bilog_list(zone, bucket, gen = None, args = None):
     cmd = ['bilog', 'list', '--bucket', bucket] + (args or [])
     cmd += ['--tenant', config.tenant, '--uid', user.name] if config.tenant else []
+    if gen:
+        cmd += ['--gen', gen]
```

Comment:
Yup, we can use that too. But wouldn't keeping `gen` as a separate parameter be good choice for the future tests?

Reply:
i guess it depends on what we change in the future. if we just add more arguments, it's easier to pass those in `args` too instead of changing this `bilog_list()` helper function